articles/multicollinearity/index.md

```diff
@@ -0,0 +1,131 @@
+
+As much as high VIFs are a cause of concern, there exist situations where they can be ignored. Multicollinearity will not manifest. These include:
+
+1. High VIFs in control variables. When high VIFs are present in control variables but not in variables in consideration, they can be ignored. This would mean that the variables we are interested in are neither collinear to the control variables nor each other. As such, there is no impact on the coefficients of regression $(R_i^2)$.
+
+2. Involvement of powers and products of other variables. We may have high VIFs resulting from the involvement of multiplications or products or other variables. Consider a regression model with $y$ and $y^2$ as independent variables. In this case, the resulting multicollinearity is not harmful.
+
+3. Dummy variable representing more than two categories. A dummy variable is used to represent a specific attribute with two or more clear categories. It takes the value of 0 or 1 to show the absence or presence of a given property. If a dummy variable represents more than two categories with a high VIF score, multicollinearity might not exist. If there is a fragment of cases in a given category, the variables will always give high VIF values. This is regardless of whether there is a correlation between categorical variables and other variables.
+
+### Addressing Multicollinearity
+
+If we conclude that multicollinearity poses a problem for our regression model, we can attempt a handful of basic fixes.
+
+**Removing variables.** A straightforward method of correcting multicollinearity is removing one or more variables showing a high correlation. This assists in reducing the multicollinearity linking correlated features. It is advisable to get rid of variables iteratively. We would begin with a variable with the highest VIF score since other variables are likely to capture its trend. As a result of removing such a variable, other variables' VIF values are likely to reduce.
+
+**More data.** Statistically, a regression model with more data is likely to suffer less variance due to a larger sample size. This reduces the impact of multicollinearity.
+
+**Partial least squares regression (PLS) or principal component analysis (PCA).** A takeaway from this paper on [partial least squares regression](http://www.ijastnet.com/journals/Vol_4_No_1_January_2014/17.pdf) for multicollinearity is that PLS can lessen variables to a smaller grouping with no correlation between them. PLS, like PCA, is a dimensionality reduction technique. PCA reduces the dimension of data through the decomposition of data into independent factors. Resultantly, new variables with no correlation between them are created. This [article](https://towardsdatascience.com/how-to-remove-multicollinearity-in-dataset-using-pca-4b4561c28d0b) explains how PCA handles multicollinearity.
```

Comment:
The heading needs to changed... Are you suggesting that PLS and PCA are the same? It needs to be made clear that PLS is similar to PCA... The heading is misleading.